contrib/start_rspec

```diff
@@ -1,8 +1,28 @@
 #!/bin/bash
+
+rm -f /obs/src/api/tmp/rspec_prepared
+
+# we need to play a little race prevention
+while ! test -e /obs/src/api/tmp/testapp_waiting; do
```

Comment:
You can use something like [&wait_for_database](https://github.com/openSUSE/open-build-service/blob/master/.circleci/config.yml#L38) on circle-ci and something like [healthcheck](https://github.com/openSUSE/open-build-service/blob/master/docker-compose.ahm.yml#L18) in docker-compose for all of this.

Reply:
Actually we wouldn't need all of this if you told me how I can make the rspec container join the default network when using d-c run - it only does so on 'up'. Capybara can start a server (not webrick though, but puma worked fine and I'm sure we can make unicorn work too)